content/integrations/faq/troubleshooting-jmx-integrations.md

```diff
@@ -87,6 +87,19 @@ Note: the location to the JRE tools.jar (`/usr/lib/jvm/java-8-oracle/lib/tools.j
 
 **Note**: you may wish to specify alternative JVM heap parameters `-Xmx`, `-Xms`, the values used in the example correspond to the JMXFetch defaults.
 
+## Agent v5 troubleshooting
+
+
+| Command                                                           | Description                                                                                                                                                             |
+| :----------------------------------------                         | :---                                                                                                                                                                    |
+| `sudo /etc/init.d/datadog-agent jmx list_matching_attributes`     | List attributes that match at least one of your instances configuration.                                                                                                |
```

Comment:
should it be "List attributes that match at least one of your instance configurations"? I'm not completely sure what this means.

Reply:
It's the way the DD JMX integration is working, `attributes` and `instances`are referring parameters for JMX integrations: https://github.com/DataDog/integrations-core/blob/master/cassandra/datadog_checks/cassandra/data/conf.yaml.example